assets/src/dashboard/icons/left-arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M8.00009 15.3332L9.29259 14.0407L4.17758 8.91651L15.3334 8.91651L15.3334 7.08317L4.17758 7.08317L9.30175 1.96817L8.00008 0.666505L0.666749 7.99984L8.00009 15.3332Z" fill="#3F454A"/>
```

Comment:
I think we've been trying to use `current-color` as the fill for svgs

Reply:
Oh, good catch, these are my first SVG imports 😱, I'll update them both!